test/unit/ItemsContainer.test.js

```diff
@@ -317,7 +313,7 @@ describe('ItemsContainer', () => {
     after(() => {
       component.unmount()
     })
-    it(`should have ${itemsListPageLimit} on the first page of the item table and ${longListItems.length - itemsListPageLimit} on the second`, () => {
+    xit(`should have ${itemsListPageLimit} on the first page of the item table and ${longListItems.length - itemsListPageLimit} on the second`, () => {
```

Comment:
Yes, this can be deleted. We are now just fetching and displaying 20 items at a time with the pagination update.